website/templates/website/documentation_page.html

```diff
@@ -1,6 +1,15 @@
 {% extends "website/base_template.html" %}
 {% load customfilters %}
 
+{% block extra_css %}
+<style type="text/css">
+	.section {
+    padding-top: 50px;
```

Comment:
@jchoude I could not find any better solution actually. It works for different screen sizes. But I just found a problem. The breadcrum links at the top of the docs can't be clicked.